src/components/KeyAndTuningButton.tsx

```diff
@@ -1,19 +1,40 @@
-import { useState } from 'react';
-import Stack from '@mui/material/Stack';
+import React, { useState } from 'react';
 import Button from '@mui/material/Button';
+import { Box } from '@mui/material';
 
-export default function BasicButtons() {
+interface RootOption {
+  value: string;
+  label: string;
+}
+
+interface KeyAndTuningButtonProps {
+  isIncomplete: boolean;
+  selectedRoot: RootOption | null;
+  selectedMode: string | null;
+  selectedTuning: string | null;
+}
+
+const KeyAndTuningButton: React.FC<KeyAndTuningButtonProps> = ({
+  isIncomplete,
+  selectedRoot,
+  selectedMode,
+  selectedTuning
+}) => {
   const [isClicked, setClicked] = useState(false);
-  const [isIncomplete, setIncomplete] = useState(false); //will be true if note, mode, and tuning are not complete
 
   const handleClick = () => {
-      setClicked(true)
+    setClicked(true);
+    console.log("Selected root: ", selectedRoot);
+    console.log("Selected mode: ", selectedMode);
+    console.log("Selected tuning: ", selectedTuning);
   };
+
   return (
-    <Stack spacing={2} direction="row" color="black">
+    <Box sx={{ maxWidth: 450, display: 'flex', justifyContent: 'left' }}>
       <Button 
         onClick={handleClick}

```

Comment:
@stenzeladam instead of creating a `handleClick` method, you can just do this:

```onClick={() => setClicked(true)}```

Reply:
But the `handleClick` method will also be used to at least create a `Mode` object, where that `Mode` instance will be vital to creating the corresponding `Chord` objects. It should make more sense in newer versions of the code. For example:

```
const handleClick = () => {
    setClicked(true);
    console.log("Selected root: ", selectedRoot?.value);
    console.log("Selected mode: ", selectedMode);
    console.log("Selected tuning: ", selectedTuning);
    if (selectedRoot && selectedMode) {
      let instance = new Modes(selectedRoot.value);
      instance.applyMode(selectedMode);
      let tempChord = new Chord(1, instance.getScale(), instance.getChromatic());
      tempChord.buildChord();
      console.log(tempChord.getNotes_String());
    }
  };
```